docs/source/tutorials_source/platform/tutorial_label_studio_export.py

```diff
@@ -0,0 +1,78 @@
+"""
+
+.. _lightly-tutorial-export-labelstudio:
+
+Tutorial 10: Export to LabelStudio
+=============================================
+
+This tutorial shows how you can easily label all images of a tag from Lightly
+using the open-source data labeling tool `LabelStudio <https://labelstud.io>`_.
+
+What you will learn
+--------------------
+
+* Export a tag from Lightly in the `LabelStudio format <https://labelstud.io/guide/tasks.html#Basic-Label-Studio-JSON-format>`_.
+* Import the tag into LabelStudio.
+
+Requirements
+------------
+You have a dataset in the `Lightly Platform <https://app.lightly.ai>`_
+and optionally already chosen a subset of it and created a tag for it.
+Now you want to label all images of this tag using LabelStudio.
+
+If you have not created your own dataset yet, you can use any dataset
+(e.g. the playground dataset) or follow one of the other tutorials to create one.
+The :ref:`lightly-tutorial-sunflowers`
+is particularly well-suited.
+
+Launch LabelStudio
+------------------
+Follow the documentation to `install and start Labelstudio <https://labelstud.io/guide/index.html#Quick-start>`_.
+Then create a new project and click on import. Now you should be in the
+import screen.
+
+.. figure:: ../../tutorials_source/platform/images/tutorial_export_labelstudio/labelstudio_import_dialog.jpg
+    :align: center
+    :alt: Import dialog of LabelStudio.
+
+Export from Lightly in the LabelStudio format
+---------------------------------------------
+Now open your dataset in the `Lightly Platform <https://app.lightly.ai>`_.
```

Comment:
Last wish :) but will approve already
can you make this as a list of steps to do with `-`? Otherwise its merged on to a consecutive line of text